lisa/sut_orchestrator/azure/common.py

```diff
@@ -1049,15 +1051,47 @@ def wait_copy_blob(
     timeout: int = 60 * 60,
 ) -> None:
     log.info(f"copying vhd: {vhd_path}")
-    check_till_timeout(
-        lambda: blob_client.get_blob_properties().copy.status == "success",
-        timeout_message=f"copying VHD: {vhd_path}",
-        timeout=timeout,
-        interval=2,
-    )
+    if blob_client.get_blob_properties().copy.status:
+        check_till_timeout(
+            lambda: blob_client.get_blob_properties().copy.status == "success",
+            timeout_message=f"copying VHD: {vhd_path}",
+            timeout=timeout,
+            interval=2,
+        )
+    else:
+        # If the blob is copied by AzCopy, the copy.status is None.
+        # Confirm the copy operation is success by checking the metadata.
+        check_till_timeout(
+            lambda: blob_client.get_blob_properties().metadata.get("AzCopyStatus", None)
+            == "Success",
+            timeout_message=f"copying VHD: {vhd_path}",
+            timeout=timeout,
+            interval=2,
+        )
     log.info("vhd copied")
 
 
+def copy_blob_by_azcopy(
+    azcopy_path: str,
+    src: str,
+    dst: str,
+    log: Logger,
+    timeout: int = 60 * 60,
+) -> None:
+    try:
+        subprocess.check_output(
```

Comment:
use local node, so the log can be included in normal format.

Reply:
Used local().execute() and deleted the new added function copy_blob_by_azcopy